_api-reference/security/authentication/index.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Authentication APIs
+parent: Security APIs
+has_children: true
+nav_order: 10
+---
+
+# Authentication APIs
+
+Authentication is a fundamental aspect of security in OpenSearch, verifying the identity of users and services before granting access to protected resources. The Security plugin provides several APIs to manage authentication, obtain user information, and handle credentials.
+
+## Available APIs
+
+The Authentication APIs include the following operations:
+
+- [Authentication Information API]({{site.url}}{{site.baseurl}}/api-reference/security/authentication/auth-info/): Returns information about the currently authenticated user, including roles, backend roles, and tenant memberships. Useful for debugging authentication issues, verifying permissions, and retrieving user context for applications.
+
+- [Authentication Token API]({{site.url}}{{site.baseurl}}/api-reference/security/authentication/auth-token/): Generates authentication tokens for use in place of basic authentication. Ideal for applications and scripts that require secure access without storing plaintext credentials.
+
+- [Change Password API]({{site.url}}{{site.baseurl}}/api-reference/security/authentication/change-password/): Lets users update their own passwords securely. Requires verification of the current password and does not require administrator involvement.
+
+## Authentication workflows
+
+These APIs support the following common authentication workflows:
+
+- **User verification**: Confirm a user’s identity and permissions before executing sensitive operations.
+- **Service-to-service communication**: Use tokens to enable secure communication between applications and OpenSearch.
```

Comment:
While this is a great line to include in the documentation, its not applicable for the authtoken endpoint above. The authtoken endpoint is specific to SAML auth and is used for an OpenSearch Dashboards session to prevent roundtrips back to the SAML IdP during an active session.

Reply:
I'll remove it.